### DISCUSSION TITLE
Demo site and documentation

### DISCUSSION THREAD

**Discussion by Andras-Csanyi** [microsoft/fluentui-blazor#286] [General]
Hey all,

My question is related to the demo site. I know we have one, but that is not clear for me whether it is refreshed by every release (or every commit to the main). I see that this steps fails with my PRs and I assume that is the right thing, but I haven't followed up what should happen.

What are the plans with the demo site and documentation? I see that it can be enhanced.
Right now I'm working on breadcrumb tests and there is this `download` attribute. It is not mentioned either on fast.design, nor in the demo project. Since I've been going through the details of the components I also can double check whether the documentation is accurate and actualised or not.

**Comment by vnbaaij** [comment]
Yes, the demo site is refreshed as part of the CI build that runs on every commit.
A new copy of the site is created when a PR is created but you don't have enough rights to do so. That is why you are seeing the error.

My plan is to keep the demo up to date, showing as many capabilities of the library as possible. As the demo is part of the repo (in the `examples` folder) you can just create PR's if there are things you ok need to be changed.

The component's documentation is generated by using reflection and should therefore always be accurate and up-to-date.

**Reply by Andras-Csanyi** [reply]
Thank you for the answer!

**Reply by osmanium** [reply]
Would it be also possible to publish a new VNext nuget package? :) I see there are a lot of new stuff but last version (3.0.0-preview.3) is 2 months old, I'd appriciate very much something like night build

**Reply by vnbaaij** [reply]
A new version of v3 NuGet is on it's way (later today)!

We are changing the build pipeline but not sure yet if we will do nightly releases as that would drain our allowed DevOps build capacity probably too quickly
